src/maker/config.rs

```diff
@@ -32,20 +34,94 @@ pub struct MakerConfig {
 impl MakerConfig {
     /// Init a default configuration with given port and address
     pub fn init(port: u16, onion_addrs: String) -> Self {
```

Comment:
`port` and `onion` should not be a parameter. It was used as a "easy way". They should also be included as part of the conf file. With default values..

The onion address should not be part of the conf, as it should be a system-generated random address. But that can be handled later once we do the Tor integration. For now its even okay to hardcode this sample onion address with a TODO comment. It's not used anywhere.